### PR TITLE
[scenes] Add scenes to test several components during mesh removal

### PR DESCRIPTION
**Pull request by epernod**
It would be better as unit test.. (when I have time), but until then, I put those scenes inside examples/benchmarks/TopologicalChanges

NB: some scenes are crashing. This reveals some component have been created with a copy past of the TopologyHandler but are not really handling topological changes. 

______________________________________________________

By submitting this pull request, I acknowledge that  
**I have read, understand, and agree [SOFA Developer Certificate of Origin (DCO)](https://github.com/sofa-framework/sofa/blob/master/CONTRIBUTING.md#sofa-developer-certificate-of-origin-dco)**.
______________________________________________________

**Reviewers will merge this pull-request only if**  
- it builds with SUCCESS for all platforms on the CI.
- it does not generate new warnings.
- it does not generate new unit test failures.
- it does not generate new scene test failures.
- it does not break API compatibility.
- it is more than 1 week old (or has fast-merge label).

### CODE REVIEW

**Comment by epernod** [comment]
[ci-build][with-all-tests]

File: examples/Benchmark/TopologicalChanges/AffineMovementConstraint__RemovingMeshTest.scn [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" ?>
+<!-- Automatic Triangle removal on a simple Triangle topology with FEM: Element removed are define in: ./RemovingTrianglesProcess.txt -->
+<Node name="root" gravity="0 -9 0" dt="0.01">
+    <RequiredPlugin pluginName="SofaOpenglVisual"/>
```
Comment (alxbilger) [inline examples/Benchmark/TopologicalChanges/AffineMovementConstraint__RemovingMeshTest.scn]: To please @guparan :
```suggestion
    <RequiredPlugin name="SofaOpenglVisual"/>
```

**Comment by fredroy** [comment]
Indeed :
 - SceneTests.examples/Benchmark/TopologicalChanges.ProjectToPlaneConstraint_RemovingMeshTest.scn
 - SceneTests.examples/Benchmark/TopologicalChanges.FixedPlaneConstraint_RemovingMeshTest.scn
crash.
Should we let them crash and appear in the CI log (thus maybe making the result even more confusing), or should we ignore it on the CI ?

**Comment by epernod** [comment]
yes as I said in the description, they are failing because the code is not working. 
Either we put them to remind me I need to fix those components (but could be in several months)
or we remove them from PR and copy past in an issue (maybe cleaner) ?

**Comment by epernod** [comment]
> 
> 
> Indeed :
> 
>     * SceneTests.examples/Benchmark/TopologicalChanges.ProjectToPlaneConstraint_RemovingMeshTest.scn
> 
>     * SceneTests.examples/Benchmark/TopologicalChanges.FixedPlaneConstraint_RemovingMeshTest.scn
>       crash.
>       Should we let them crash and appear in the CI log (thus maybe making the result even more confusing), or should we ignore it on the CI ?

yes as I said in the description, they are failing because the code is not working.
Either we put them to remind me I need to fix those components (but could be in several months)
or we remove them from PR and copy past in an issue (maybe cleaner) ?

**Comment by fredroy** [comment]
Yeah just I wanted to write which ones were failing.
As for your question.... I suppose the issue thingy is cleaner, but dont remove the tests themselves (just ignore it in the list with a comment with a link to the issue)

**Comment by epernod** [comment]
[ci-build][with-all-tests]

**Review by fredroy — APPROVED** [review]

**Comment by fredroy** [comment]
For the ignoring part,apparently you have to set the 2 ignore lines into examples/.scene-tests ( and not into examples/Benchmark/TopologicalChanges/.scene-tests )

**Comment by epernod** [comment]
> 
> 
> @epernod this scene crashes for me : `FixedPlaneConstraint_RemovingMeshTest.scn`

yes see linked issue: #2189

File: examples/.scene-tests [hunk 2]
```diff
@@ -1,42 +1,48 @@
-# This runs very slowly, so compute only a few iterations
+# [Slow] This runs very slowly, so compute only a few iterations
```
Comment (fredroy) [inline examples/.scene-tests:1]: Nice idea to tag on the ignored/timeouted/whatever scenes

**Review by fredroy — APPROVED** [review]

**Pull request merged**